### PR TITLE
Added a note about a k3s issue wrt iptables

### PR DESCRIPTION
**Pull request by e-minguez**
Signed-off-by: Eduardo Minguez <e.minguez@gmail.com>

## Description
Related to #89 

## Motivation and Context
Warn users to a possible issue wrt network not working.
- [x] I have raised an issue to propose this change

## How Has This Been Tested?
Deployed k3s in Debian Buster and facing the issue.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

None of those.

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ export KUBECONFIG=`pwd`/kubeconfig
 kubectl get node
 ```
 
+> Note: There is an issue in k3s involving iptables that can affect the network communication. See the [k3s issue](https://github.com/rancher/k3s/issues/703) and the corresponding [kubernetes one](https://github.com/kubernetes/kubernetes/issues/71305) for more information and workarounds.
```
Comment (alexellis) [inline README.md]: This should mention the OS that it was observed with

Comment (e-minguez) [reply README.md]: :+1:

**Comment by e-minguez** [comment]
> Added a comment. I think this could go in the appendix too.

There is no appendix AFAIK. I've added the Debian Buster clarification.

**Comment by alexellis** [comment]
Let's create an appendix right at the bottom of the file

**Comment by e-minguez** [comment]
> Let's create an appendix right at the bottom of the file

Done

**Comment by e-minguez** [comment]
Thank you!

**Pull request merged**